packages/gatsby-remark-code-repls/README.md

```diff
@@ -0,0 +1,42 @@
+# gatsby-remark-code-repls
+
+Gatsby plugin to auto-generate links to popular REPLs like [Babel](https://babeljs.io/repl/) and [Codepen](https://codepen.io/).
+
+This plug-in reads source code that is stored alongside your Gatsby site and passes it to the specified REPL when a user clicks the generated link.
+
+## Install
+
+`npm install --save gatsby-remark-code-repls`
+
+## How to Use
+
+```javascript
+// In your gatsby-config.js
+{
+  resolve: 'gatsby-remark-code-repls',
+  options: {
+    // Optional default link text.
+    // eg <a href="...">Click here</a>
+    defaultText: 'Click here',
+
+    // Example code links are relative to this dir.
+    // eg examples/path/to/file.js
+    directory: `${__dirname}/examples/`,
+
+    // Optional additional externals to load from CDN.
```

Comment:
Perhaps instead of making this "additional" to react/react-dom, make those the defaults? Non-react people might use this plugin e.g. someone building a site for a JS lib.

Reply:
Yeah, I had considered that option too. I don't feel strongly about the one I chose, so I'm happy to change it.

Reply:
I think maybe it's best to just remove the defaults entirely. 😄 I'll do that.